geonode/harvesting/harvesters/base.py

```diff
@@ -173,6 +173,16 @@ def update_geonode_resource(
                     f"the remote resource {harvested_info.resource_descriptor.uuid!r}")
             geonode_resource = resource_manager.update(
                 str(harvested_info.resource_descriptor.uuid), vals=defaults)
+
+        # update keywords and regions
```

Comment:
Please remove this comment, as it is pretty clear what the code is doing

Reply:
updatd @ricardogsilva